### PR TITLE
Update 01-nkatangoor.md

### PR DESCRIPTION
**Pull request by nkatangoor**
## Adding an Image
This is an image

### CODE REVIEW

File: _slides/01-nkatangoor.md [hunk 1]
```diff
@@ -4,3 +4,4 @@ title: "Welcome to our slide deck!"
 ---
 
 Use the right arrow to begin!
```
Comment (evgenyrahman) [inline _slides/01-nkatangoor.md:6]: @nkatangoor Thank you for your comment, can you add more content?

File: _slides/01-nkatangoor.md [hunk 2]
```diff
@@ -4,3 +4,4 @@ title: "Welcome to our slide deck!"
 ---
```
Comment (evgenyrahman) [inline _slides/01-nkatangoor.md:4]: This should be four dashes

**Review by evgenyrahman — CHANGES_REQUESTED** [review]

File: _slides/01-nkatangoor.md (hunk 1 above)
Comment (nkatangoor) [reply _slides/01-nkatangoor.md:6]: @evgenyrahman Sure will add more!

**Review by evgenyrahman — APPROVED** [review]
👍  Looks good to me

**Review by katiem0 — APPROVED** [review]

**Pull request merged**